src/Header/Header.test.tsx

```diff
@@ -1,22 +1,55 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
+import { render, screen, fireEvent, getByAltText } from "@testing-library/react";
```

Comment:
Just a heads up you dont have to import in getByAllText it lives on screen

Reply:
what!!! maybe I was thinking of another time when Node yelled at me